src/AspNetCoreModuleV2/InProcessRequestHandler/managedexports.cpp

```diff
@@ -469,6 +472,8 @@ EXTERN_C __MIDL_DECLSPEC_DLLEXPORT
 VOID
 set_main_handler(_In_ hostfxr_main_fn main)
 {
+    // Allow inprocess application to be recreated as we reuse the same CLR
+    g_fInProcessApplicationCreated = false;
```

Comment:
We will need to be careful here with the test server. Technically this isn't the right place to add it, as the next request will create a shutdown application

Reply:
> as the next request will create a shutdown application?

Why? We don't shutdown test server in the middle of the test.

Reply:
I see now. This makes sense.